lib/vanagon/platform/dsl.rb

```diff
@@ -51,6 +51,7 @@ def platform(name, &block)
                     end
 
         yield(self)
+        environment 'VANAGON_PLATFORM', platform_name.tr('-', '.')
```

Comment:
should we remove the `-` at a later point in time, since I think that's only really needed when we send metrics. Would it be confusing to have this variable be in the format `el.7.x86_64` rather than `el-7-x86_64`?